io/config.go

```diff
@@ -64,6 +65,8 @@ func (config *Config) Read() {
 	if err != nil {
 		error_handler.ErrorExit(err)
 	}
+	defer fileToRead.Close()
```

Comment:
The same as above file; `os.OpenFile` returns file, nil or nil, err, but not both file object and error.